src/views/PipelineView.vue

```diff
@@ -40,23 +45,15 @@ export default {
     path(route) {
       return '/' + this.pipelineInfo.name + '/' + route
     },
-    /** Callback handler for tab item selection - Each tab selection navigates user to a different page which
-     * is handled by the router.push method**/
-    setSource(index) {
-      this.sourceIndex = index
-      // If new tab is already active, do nothing
-      if (this.isActive(this.pipelineInfo.routes[index])) {
-        return
-      }
-      this.$router.push({ path: this.path(this.pipelineInfo.routes[index]) }).catch((error) => {
-        if (!isNavigationFailure(error, NavigationFailureType.duplicated)) {
-          console.error(error)
-        }
-      })
-    },
```

Comment:
Just to understand - Is this not required anymore? How are we dealing with navigation failures now?

Reply:
We are not using router-links instead of manipulating the router paths ourselves so we don't need this setSource function.